src/v2/Components/Artwork/SaveButton/NewSaveButton.tsx

```diff
@@ -0,0 +1,103 @@
+import { AuthContextModule } from "@artsy/cohesion"
+import { NewSaveButton_artwork } from "v2/__generated__/NewSaveButton_artwork.graphql"
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { HeartIcon, Clickable, HeartFillIcon } from "@artsy/palette"
+import { useSaveArtwork } from "./useSaveArtwork"
+import { useTracking } from "v2/System"
+import { useState } from "react"
+
+export interface SaveTrackingProps {
+  context_page?: string
+}
+
+export interface SaveButtonProps {
+  artwork: NewSaveButton_artwork
+  contextModule: AuthContextModule
+}
+
+const BTN_HEIGHT = 20
+const BTN_WIDTH = 20
+
+export const NewSaveButton: React.FC<SaveButtonProps> = ({
+  artwork,
+  contextModule,
+}) => {
+  const tracking = useTracking()
+  const [isHovered, setIsHovered] = useState(false)
```

Comment:
**question**: maybe we can use [useSaveButton](https://github.com/artsy/force/blob/7004af4c956c28111ef4910695cd9c0f82683ca9/src/v2/Components/Artwork/SaveButton/useSaveButton.tsx#L10) hook here?

Reply:
Why would we want to do that though? 🤔

Reply:
To be able to reuse [this logic](https://github.com/artsy/force/blob/7004af4c956c28111ef4910695cd9c0f82683ca9/src/v2/Components/Artwork/SaveButton/useSaveButton.tsx#L17-L27) [here](https://github.com/artsy/force/blob/7004af4c956c28111ef4910695cd9c0f82683ca9/src/v2/Components/Artwork/SaveButton/NewSaveButton.tsx#L52-L58) and [here](https://github.com/artsy/force/blob/7004af4c956c28111ef4910695cd9c0f82683ca9/src/v2/Components/Artwork/SaveButton/NewSaveButton.tsx#L27). It's just a suggestion, if you don't agree with it – just ignore it. Maybe I get too hung up on such things😅

Reply:
Cool, gave it a quick try and i felt it complicated things 🙈  will ignore it for now but thanks for raising that!

Reply:
Got it. Thanks!